# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/transforms/big-axis-aligned-scale.yaml
# Commit: b7f88e5c537b
# Full Hash: b7f88e5c537bb7d64ba9d4cb58b052650d510638
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-08 17:32:27
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Improve how WR handles bounding rects for off-screen surfaces r=gfx-reviewers,kvark
#   
#   This patch introduces a number of subtle but important changes
#   to how we deal with off-screen surfaces. The overall goals are:
#   
# ==============================================================================

diff -r 8e9c1ffdeed8 -r b7f88e5c537b gfx/wr/wrench/reftests/transforms/big-axis-aligned-scale.yaml
--- a/gfx/wr/wrench/reftests/transforms/big-axis-aligned-scale.yaml	Mon Feb 07 22:02:35 2022 +0000
+++ b/gfx/wr/wrench/reftests/transforms/big-axis-aligned-scale.yaml	Mon Feb 07 22:14:47 2022 +0000
@@ -3,7 +3,7 @@
   items:
     - type: stacking-context
       transform-style: preserve-3d
-      transform: scale(-2, 44, 44727)
+      transform: scale(-2, 44, 1)
       items:
         - type: rect
           bounds: [ -100, -100, 200, 101 ]